pyxem/signals/beam_shift.py

```diff
@@ -18,17 +18,19 @@
 
 
 import numpy as np
-import pyxem.utils._pixelated_stem_tools as pst
+from scipy.ndimage import rotate
+import pyxem.utils._beam_shift_tools as bst
 from hyperspy._signals.lazy import LazySignal
-from hyperspy._signals.signal1d import Signal1D
+from hyperspy.signals import Signal1D
+from hyperspy.utils.plot import plot_images
 
 
 class BeamShift(Signal1D):
```

Comment:
@magnunor is it okay if I make a couple of adjustments to this?  I think we should make BeamShift a subclass of Vector1D.  I'm planning on moving some of the plotting capablilites to that class and then that can be a more general feature. I think the other place where that might be useful is with the `StrainMap` class which is essentially plotting 4 Vector quantities.

Reply:
Go for it!